onnx_chainer/export.py

```diff
@@ -203,6 +196,10 @@ def export(model, args, filename=None, export_params=True,
         train (bool): If True, output computational graph with train mode.
         return_flat_inout (bool): If set True, return ONNX model with flat
             inputs, and flat outputs.
+        external_converters (dict): Add-on converter. Convert functions
```

Comment:
Do you think we can keep these APIs compatible? I'm afraid we may eventually need to change them. If you agree somewhat, how about mentioning these two APIs may be changed in future? If we'll be comfortable with them, we can drop the scary "this API is experimental" comment after, say, a half year?

Reply:
add warning when `external_converters`, `external_imports` or `return_flat_inout` is used, thx for suggestion.